c7n/resources/awslambda.py

```diff
@@ -305,6 +307,60 @@ def process(self, resources, event=None):
 
         return results
 
+@AWSLambda.action_registry.register('update')
+class UpdateLambda(Action):
+    """Update a lambda's configuration.
+
+    This action also has specific support for enacting recommendations
+    from the AWS Cost Recommendation Hub for resizing.
+
+    :example:
+
+      .. code-block:: yaml
+
+         policies:
+           - name: lambda-gc
+             resource: aws.lambda
+             filters:
+               - type: cost-recommendation
```

Comment:
```suggestion
               - type: cost-optimization
```

Reply:
doh. thanks, also going to change the doc policy name